src/components/backup/RestoreCloudStep.js

```diff
@@ -155,11 +155,16 @@ export default function RestoreCloudStep({
 
   const onSubmit = useCallback(async () => {
     try {
+      Keyboard.dismiss();
       setIsWalletLoading(WalletLoadingStates.RESTORING_WALLET);
       const success = await restoreCloudBackup(
```

Comment:
Hm, maybe if there's a need to add comments to explain which param is which (L164 and L166), then maybe it makes sense to make this function take config object so then params would be named here?

```js
const success = await restoreCloudBackup({
  password,
  userData,
  name: backupSelected?.name,
  onBeforePINCreated: () => setIsWalletLoading(null),
  onAfterPINCreated: () => setIsWalletLoading(WalletLoadingStates.RESTORING_WALLET)
});
```

To me the variant above looks more readable.

Reply:
Yeah, it totally makes sense, thanks for the tip